lib/mix/tasks/phx.gen.context.ex

```diff
@@ -350,7 +358,25 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     end
   end
 
-  defp validate_args!([context, schema, _plural | _] = args, help) do
+  defp validate_args!(
+         [maybe_context_name, schema_name_or_plural, plural_or_first_attr | schema_args],
+         optional,
+         help
+       ) do
+    has_context? =
+      case schema_name_or_plural do
+        <<char::integer-size(8), _rest::binary>> when char in ?A..?Z -> true
```

Comment:
That's the default, in case you want to skip it.

```suggestion
        <<char, _rest::binary>> when char in ?A..?Z -> true
```